src/main/java/com/raoulvdberge/refinedstorage/api/storage/IStorageCacheListener.java

```diff
@@ -25,4 +29,10 @@
      * @param size  the size, negative if the amount decreases
      */
     void onChanged(@Nonnull T stack, int size);
+
+    default void onChangedBulk(@Nonnull List<Pair<T, Integer>> stacks)
+    {
```

Comment:
Coding style

Reply:
Feel free to break BC

Reply:
Sorry, I have my editor set to correct brace positioning, I'll fix it. ;)

As for BC, do you mean to just replace `onChanged` entirely? I only didn't do that because I wasn't sure if you expect any external implementations.

Reply:
You can remove the default method.